charts/atlas-operator/templates/cluster-roles.yaml

```diff
@@ -9,7 +9,10 @@ metadata:
   labels:
   {{- include "mongodb-atlas-operator.labels" $ | nindent 4 }}
 rules:
-{{ template "mongodb-atlas-operator.rbacRules" $ | toYaml | indent 2 }}
+{{- range (.Files.Lines "rbac.yaml") }}
+  {{ . -}}
+{{- end }}
+{{/*{{ template "mongodb-atlas-operator.rbacRules" $ | toYaml | indent 2 }}*/}}
```

Comment:
Is this line a comment? can we remove it?

Reply:
Thanks! Fixed